content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -2906,6 +2933,11 @@ The `progressbar` element can have the following attributes:
    :type: str
    :default: `''`
 
+`control`: customize create and delete actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Like for :ref:`list views <reference/view_architectures/list/control>`.
+
 .. admonition:: Possible structure and representation of its rendering
 
    .. list-table::
```

Comment:
This is specific to the `progressbar` element. The new `control` section should come after.